subsys/bluetooth/host/conn.c

```diff
@@ -2390,6 +2390,11 @@ struct bt_conn *bt_conn_lookup_addr_br(const bt_addr_t *peer)
 {
 	int i;
 
+	CHECKIF(peer == NULL) {
+		LOG_WRN("Invalid peer address");
```

Comment:
```suggestion
	if (peer == NULL) {
		LOG_DBG("Invalid peer address");
```

Based on the most recent discussion in the BT WG meeting, I think we should work towards no using CHECKIF anymore. 

Similar comment to your other PR about using LOG_WRN -> LOG_DBG for APIs that also return an error